draft-ietf-tls-dtls-rrc.md

```diff
@@ -152,6 +153,55 @@ cookie is a 8-byte field containing arbitrary data.
 The `return_routability_check` message MUST be authenticated and encrypted using
 the currently active security context.
 
+# Off-Path Packet Forwarding {#off-path}
+
+An off-path attacker that can observe packets might forward copies of
+genuine packets to endpoints. If the copied packet arrives before
+the genuine packet, this will appear as a NAT rebinding. Any genuine
+packet will be discarded as a duplicate. If the attacker is able to
+continue forwarding packets, it might be able to cause migration to a
+path via the attacker. This places the attacker on-path, giving it
+the ability to observe or drop all subsequent packets.
+
+This style of attack relies on the attacker using a path that has
+approximately the same characteristics as the direct path between
+endpoints. The attack is more reliable if relatively few packets are
+sent or if packet loss coincides with the attempted attack.
+
+A non-path probing packet received on the original path that increases the
+maximum received packet number will cause the endpoint to move back
+to that path. Eliciting packets on this path increases the
+likelihood that the attack is unsuccessful. Therefore, mitigation of
+this attack relies on triggering the exchange of packets.
+
+In response to an apparent migration, endpoints MUST validate the
+previously active path using a `return_routability_check`  exchange.
+This induces the sending of new packets on that path. If the path is
+no longer viable, the validation attempt will time out and fail;
+if the path is viable but no longer desired, the validation will
+succeed but only results in probing packets being sent on the path.
+
+An endpoint that receives a `return_routability_check` message
+containing a path_challenge on an active path SHOULD send 
+a `return_routability_check` of msg-type no_op in response. 
+If the `return_routability_check` with a `no_op` arrives before any
+copy made by an attacker, this results in the connection being
+migrate back to the original path. Any subsequent migration to
+another path restarts this entire process.
```

Comment:
I am probably misunderstanding the mechanics here.  But I'd have thought that if I receive a path_challenge on a path that is active but I don't want to use any more, I should state that explicitly using no_op (which I'd rather call path_drop)?